### DISCUSSION TITLE
Improved FORMAT spec for Vampires Empire (Digitek Inc.)

### DISCUSSION THREAD

**Discussion by SidTheMole** [keirf/disk-utilities#201] [Ideas]
The current d-a format spec for Vampires Empire (Digitek Inc.) gives 50% track errors CMD output although the result IPF is ok.

A correct format assignment would be:

"Vampires Empire"
    0-103 amigados vampires_empire_a vampires_empire_b vampires_empire_c
    104 ignore
    105 amigados vampires_empire_b
    * ignore
 
* Suggestion for future inclusion in d-a build.

**Comment by keirf** [comment]
Thanks. I will let @kkrellwitz comment on this one, since he defined the original format in d-a.

**Comment by kkrellwitz** [comment]
The format has been updated.  Not identical to the format above as I don't have a copy that has a amigados track on 105.

"Vampires Empire"
    0-103 amigados vampires_empire_a vampires_empire_b vampires_empire_c
    105 vampires_empire_b
    * ignore
